pages/staking.staking.tsx

```diff
@@ -76,23 +103,22 @@ export default function Staking() {
     });
   }, [trackEvent]);
 
-  // Total funds at Safety Module (stkaave tvl + stkbpt tvl)
   const tvl = formatUnits(
-    BigNumber.from(stakeGeneralResult?.aave.stakeTokenTotalSupply || '0')
-      .mul(stakeGeneralResult?.aave.stakeTokenPriceEth || '0')
+    BigNumber.from(stkAave?.stakeTokenTotalSupply || '0')
+      .mul(stkAave?.stakeTokenPriceUSD || '0')
       .add(
-        BigNumber.from(stakeGeneralResult?.bpt.stakeTokenTotalSupply || '0').mul(
-          stakeGeneralResult?.bpt.stakeTokenPriceEth || '0'
-        )
+        BigNumber.from(stkBpt?.stakeTokenTotalSupply || '0').mul(stkBpt?.stakeTokenPriceUSD || '0')
       )
-      .mul(stakeGeneralResult?.ethPriceUsd || 1),
-    18 + 18 + 8 // 2x total supply (18 decimals), 1x ethPriceUSD (8 decimals)
+      .add(
+        BigNumber.from(stkGho?.stakeTokenTotalSupply || '0').mul(stkGho?.stakeTokenPriceUSD || '0')
+      ), // "0"
+    18 + 8
   );
 
   // Total AAVE Emissions (stkaave dps + stkbpt dps)
   const stkEmission = formatEther(
-    BigNumber.from(stakeGeneralResult?.aave.distributionPerSecond || '0')
-      .add(stakeGeneralResult?.bpt.distributionPerSecond || '0')
+    BigNumber.from(stkAave?.distributionPerSecond || '0')
+      .add(stkBpt?.distributionPerSecond || '0')
```

Comment:
we should add the stkGho emissions in here as well

Reply:
added c7a1a9f3553716232fca3f99c11e5debcb88d82a